pkg/router/routeapihelpers/validation.go

```diff
@@ -351,6 +351,8 @@ func validateCertificatePEM(certPEM string, options *x509.VerifyOptions) ([]*x50
 
 	// Reject any unsupported cert algorithms as HaProxy will refuse to start with them.
 	switch certs[0].SignatureAlgorithm {
+	case x509.SHA1WithRSA, x509.ECDSAWithSHA1:
```

Comment:
Should this be "SHA1" or "SHA-1"? I see both in use and I'm not sure which is preferred, or correct? Perhaps this details is... just a detail.